c2corg_ui/templates/article/index.html

```diff
@@ -0,0 +1,42 @@
+<%inherit file="../base.html"/>
+<%namespace file="../helpers/common.html" import="show_title"/>
+<%namespace file="../helpers/list.html" import="add_doctype_selector"/>
+
+<%block name="pagetitle"><title ng-bind="mainCtrl.page_title('Articles')">${show_title('Articles')}</title></%block>
+
+<%block name="moduleConstantsValues">
+  module.value('mapFeatureCollection', null);
+  module.value('advancedSearchFilters', {
+    'act': {'type': 'list'},
+    'cat': {'type': 'list'},
+    'atyp': {'type': 'list'},
+    'l': {'type': 'list'}
+  });
+</%block>
+
+<header class="page-main-title">
+  <h3><span translate>Articles</span>
+    <span class="badge" ng-cloak>{{resCounter}}</span>
+  </h3>
+</header>
+<section class="documents-list-section">
+  <%include file="filters.html" />
+  <div class="documents-nav-buttons text-center">
+    <button class="btn btn-default show-documents-filters-phone" translate>Filters</button>
+    ${add_doctype_selector()}
+    <button class="btn btn-default orange-btn" protected-url-btn url="${request.route_path('articles_add')}"
+            uib-tooltip="{{'Create a new article' | translate }}" tooltip-placement="right">
+      <span class="glyphicon glyphicon-plus-sign"></span>
+    </button>
+    <app-pagination></app-pagination>
+  </div>
+  <app-advanced-search document-type="articles" use-map="false"></app-advanced-search>
+</section>
+<div class="map-right">
+  <app-map app-map-advanced-search="false" app-map-show-recenter-tools="false" app-map-default-map-filter="false"></app-map>
```

Comment:
No map for the advanced search page because articles have no geometries.

As for now, the map filtering is also (temporarly) disabled for "areas" advanced search as well. See https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/area/index.html